Wrapping/Python/CMakeLists.txt

```diff
@@ -19,6 +35,24 @@ mark_as_advanced( SimpleITK_PYTHON_EGG )
 option ( SimpleITK_PYTHON_WHEEL "Add building of python wheels to the dist target." ON )
 mark_as_advanced( SimpleITK_PYTHON_WHEEL )
 
+if(SWIG_VERSION VERSION_GREATER_EQUAL "4.2.0" AND
+        Python_VERSION VERSION_GREATER_EQUAL "3.11.0") # CMake atleast 3.26
```

Comment:
Possibly change comment to cmake_minimum_required > 3.26 (referencing the macro?) not sure about this.